ipv8/test/REST/rtest/peer_interaction_test_env.py

```diff
@@ -0,0 +1,485 @@
+import ast
+import json
+import time
+import unittest
+from base64 import b64encode
+from urllib import quote
+
+from twisted.internet.defer import returnValue, inlineCallbacks
+
+from ipv8.attestation.trustchain.block import TrustChainBlock
+from ipv8.test.REST.rtest.peer_communication import GetStyleRequests, PostStyleRequests
+from ipv8.test.REST.rtest.peer_interactive_behavior import AndroidTestPeer
+from ipv8.test.REST.rtest.rest_peer_communication import HTTPGetRequester, HTTPPostRequester
+from ipv8.test.REST.rtest.test_rest_api_peer import TestPeer
+from ipv8.test.util import twisted_wrapper
+
+
+class SingleServerSetup(unittest.TestCase):
+    """AndroidTestPeer
+    Test class which defines an environment with one well-known peer. This should be extended by other subclasses,
+    which implement specific test cases.
+    """
+
+    def initialize(self, **kwargs):
```

Comment:
Please make the arguments specific with optional values.
For example:
`port=8086, interface='127.0.0.1', configuration=None` etc.